### DISCUSSION TITLE
would be great to explain parameters, e.g. comp

### DISCUSSION THREAD

**Discussion by kerm1t** [nothings/stb#1253] [Q&A] [answered]
https://github.com/nothings/stb/blob/af1a5bc352164740c1cc1354942b1c6b72eacb8a/stb_image_write.h#L47

**Comment by nothings** [comment, marked as answer]
you mean like https://github.com/nothings/stb/blob/af1a5bc352164740c1cc1354942b1c6b72eacb8a/stb_image_write.h#L79 ?
